lighthouse-extension/package.json

```diff
@@ -22,6 +23,7 @@
     "gulp-util": "^3.0.7",
     "gulp-zip": "^3.2.0",
     "package-json-versionify": "^1.0.4",
+    "puppeteer": "^1.1.1",
```

Comment:
what do folks think about moving puppeteer up to a root-level dev dependency? As I was testing viewer yesterday I was thinking it would be great to have a similar (but should be much simpler) e2e test for viewer, and it seems a bit silly to have a copy here and in viewer when `yarn install-all` would install both anyway...

Reply:
Great idea. Viewer UI testing is the perfect use case for 🤹‍♀️

Reply:
sgtm